dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
My specific use case is with Electron, importing with `<script>`. Since it has node require it won't export to global and there's no way to access the library.